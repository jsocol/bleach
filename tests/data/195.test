<div id="128"><svg><style><img/src=x onerror=alert(128)// </b>//["'`-->]]>]</div>
