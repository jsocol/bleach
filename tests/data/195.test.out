<div id="128"><svg><style></style></svg><img src="x">//["'`--&gt;]]&gt;]</div>
